### ISSUE TITLE
KeyboardShortcut "ctrl+q" bug is back.

### ISSUE THREAD

**Issue by XanoPhobious** [spicetify/spicetify-cli#1615]
### 🔍 Have you checked Spicetify.app page for your issue?

- [X] I have checked the FAQ

### 🔍 Is there already an issue for your problem?

- [X] I have checked older issues, open and closed

### ℹ Environment / Computer Info

```Markdown
- Spotify version: 1.1.83.956.gbe7ea9a7
- Spicetify version:2.9.7
```

### 📝 Description

The bug is back, I cannot share a screenshot of the error because I cannot activate the devtool.
Previously: https://github.com/spicetify/spicetify-cli/issues/1406#issue-1101328752

**Comment by ririxidev** [comment]
I guess html stuff changed and so it broke this shortcut again.

**Comment by BachoSeven** [comment]
fixed by #1617

**Comment by ririxidev** [comment]
Fixed by #1617
